.github/ISSUE_TEMPLATE/issue_report.md

```diff
@@ -0,0 +1,8 @@
+---
+name: 🐞 Internal Issue
+about: Googlers may file issues here.
+---
+<!-- DO NOT DELETE
```

Comment:
I wonder if we want to the link to https://github.com/firebase/firebase-ios-sdk/issues/new/choose here as well just in case the original description was missed?